turbo/transactions/tracing.go

```diff
@@ -83,13 +85,13 @@ func ComputeTxEnv(ctx context.Context, engine consensus.EngineReader, block *typ
 	msg, _ := txn.AsMessage(*signer, block.BaseFee(), rules)
 	if msg.FeeCap().IsZero() && engine != nil {
 		syscall := func(contract libcommon.Address, data []byte) ([]byte, error) {
-			return core.SysCallContract(contract, data, cfg, statedb, header, engine, true /* constCall */)
+			return core.SysCallContract(contract, data, cfg, statedb, block.HeaderNoCopy(), engine, true /* constCall */)
 		}
 		msg.SetIsFree(engine.IsServiceTransaction(msg.From(), syscall))
 	}
 
 	TxContext := core.NewEVMTxContext(msg)
```

Comment:
`NewEVMTxContext` returns non-pointer types - then no much profit in creating intermediate variable (it will copy anyway)